gotooltest/setup.go

```diff
@@ -112,7 +109,7 @@ func Setup(p *testscript.Params) error {
 			// that will be used.
 			return cond == runtime.Compiler, nil
 		}
-		if goVersionRegex.MatchString(cond) {
+		if testscript.GoVersionRegex.MatchString(cond) {
```

Comment:
It seems to me like gotooltest should remove the code to deal with these gc/gccgo and goX.Y conditions, as they are now handled by the main gotooltest package. Then you also remove the need to export GoVersionRegex, which isn't great as it's just an internal detail :)

Reply:
Thanks for the feedback, done.